Release Notes/511.md

```diff
@@ -16,6 +16,12 @@
 - Effect specifiers:
   - Description: The `unexpectedAfterThrowsSpecifier` node of the various effect specifiers has been removed.
   - Pull request: https://github.com/apple/swift-syntax/pull/2219
+- `SyntaxKind` removed conformance to `CaseIterable`
+  - Description: `SyntaxKind` no longer conforms to `CaseIterable` since there is no good use case to iterate over all syntax kinds. 
+  - Pull request: 
+- `IntegerLiteralExprSyntax.Radix` removed conformance to `CaseIterable`
+  - Description: `IntegerLiteralExprSyntax.Radix` no longer conforms to `CaseIterable` since there is no good use case to iterate over all radix kinds. 
+  - Pull request: 
```

Comment:
And simular here:
```suggestion
  - Pull request: https://github.com/apple/swift-syntax/pull/2292
```

Reply:
I did indeed forget to update it. Thanks for reminding me.